tests/functional/restart/54-incompatible-db/db.sqlite3

```diff
@@ -1,11 +1,11 @@
 PRAGMA foreign_keys=OFF;
 BEGIN TRANSACTION;
-CREATE TABLE suite_params(key TEXT, value TEXT, PRIMARY KEY(key));
-INSERT INTO suite_params VALUES('uuid_str','0869fe59-918f-4a32-889c-743fd9b516b9');
-INSERT INTO suite_params VALUES('run_mode','live');
-INSERT INTO suite_params VALUES('cylc_version','7.8.6-20-g5f91b');
-INSERT INTO suite_params VALUES('UTC_mode','0');
-INSERT INTO suite_params VALUES('is_held','1');
+CREATE TABLE workflow_params(key TEXT, value TEXT, PRIMARY KEY(key));
+INSERT INTO workflow_params VALUES('uuid_str','0869fe59-918f-4a32-889c-743fd9b516b9');
+INSERT INTO workflow_params VALUES('run_mode','live');
+INSERT INTO workflow_params VALUES('cylc_version','7.8.6-20-g5f91b');
+INSERT INTO workflow_params VALUES('UTC_mode','0');
+INSERT INTO workflow_params VALUES('is_held','1');
```

Comment:
This file should be reverted, as it's recreating a Cylc 7 DB

Reply:
The simple reversion works fine, it still causes the same error on attempting a restart.